test.sh

```diff
@@ -56,9 +74,16 @@ for dir in ${VERSIONS}; do
   if [ -n "${TEST_UPSTREAM}" ]; then
     if [[ -x test/run-upstream ]]; then
       VERSION=$dir test/run-upstream
+      failed_version "$?" "$dir"
     else
       echo "-> Upstream tests are not present, skipping"
     fi
   fi
   popd > /dev/null
 done
+
+if [[ "$FAILED_VERSIONS" != "" ]]; then
+    echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+    echo "Test for image ${IMAGE_NAME} FAILED in these versions ${FAILED_VERSIONS}."
+    echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
```

Comment:
We should exit the script with non-zero value, if some test fails.

Reply:
https://github.com/sclorg/container-common-scripts/pull/369/commits/fee1f6a17fc7867f6d7e9fb3d3e36c2cf1340edc